app/views/users/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<p id="notice"><%= notice %></p>
+<div class='container'>
+  <div class="row">
+    <div class="col-6">
+      <h4>ATTENDING</h4>
+      <ul>
+        <% @user.attended_events.coming_events.each do |e| %>
+          <li><%= e.title %></li> <br> 
+        <%end%>
+      </ul>
+    </div>
+    <div class="col-6">
+      <h4>ATTENDED</h4>
+        <ul>
+          <% @user.attended_events.old_events.each do |e| %>
+            <li><%= e.title %></li> <br> 
+          <%end%>
+        </ul>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h4>HOSTING</h4>
+            <ul>
+          <% @user.events.each do |e| %>
+              <% if e.date >= Date.today %>
+              <li><%= e.title %></li><br>
+              <% end %>
```

Comment:
Please refactor the logic in this file and add it to the application_helper.rb i.e. you could make a function called dates_manage or whatever you like and using it in the views instead.